### ISSUE TITLE
Using the after argument always returns an internal server error

### ISSUE THREAD

**Issue by lwaldner** [funkhaus/ql-search#6]
Running searchWP version 3.1.8 and gl-search version 1.0.0

When running a searchWP query using the after argument if the after argument is anything but an empty string the query result will always be an internal server error.

The graphql query:

```
query SearchPages($term: String!, $after: String!) {
    searchWP(first: 1, after: $after, where: { input: $term, postType: PAGE }) {
        nodes {
            ... on Page {
                id
            }
        }
        pageInfo {
            hasNextPage
            endCursor
        }
    }
}
```

The argument values:

```
{
  "term": "test",
  "after": "YXJyYXljb25uZWN0aW9uOjUyNzY="
}
```

The query result:

```
{
  "errors": [
    {
      "message": "Internal server error",
      "category": "internal",
      "locations": [
        {
          "line": 3,
          "column": 5
        }
      ],
      "path": [
        "searchWP"
      ]
    }
  ],
  "data": {
    "searchWP": null
  }
}
```

The endCursor value was taken from a previous query

Its also worth noting that the README of the github says that the searchWP plugin must be version 3.1.9+ but the newest version of the plugin available on the website is 3.1.8.

**Comment by lwaldner** [comment]
I have also tested this query using the searchWP plugin available at `https://demo.axistaylor.com/searchwp.zip` which was taken from the test scripts here `https://github.com/funkhaus/ql-search/blob/master/bin/install-wp-tests.sh#L191` and I got the same error shown above.

**Comment by lwaldner** [comment]
Tested this query with searchWP plugin version 3.1.9 and got the same error

**Comment by drewbaker** [comment]
Fixed in #7
